lms/models/lti_user.py

```diff
@@ -58,6 +59,21 @@ def from_auth_params(application_instance, lti_core_schema):
             email=lti_core_schema["lis_person_contact_email_primary"],
         )
 
+    def serialize(self) -> dict:
+        """
+        Return a dict representing the LTIUser.
+
+        LTIUser is often serialized. We can pick here the exact representation.
```

Comment:
With the change to `BearerTokenSchema` this is now the only place that controls how LTIUser is serialize.

Reply:
Centralising this seems sensible.